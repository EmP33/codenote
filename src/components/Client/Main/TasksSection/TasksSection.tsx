import React from "react";
import Tasks from "../../../elements/TasksElement/Tasks";

const TasksSection: React.FC = () => {
  return <Tasks />;
};

export default TasksSection;
