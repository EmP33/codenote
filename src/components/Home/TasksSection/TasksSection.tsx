import React from "react";
import { Container, Grid, Typography } from "@mui/material";
import styled from "styled-components";
import ContentPoint from "../ContentPoint/ContentPoint";
import Tasks from "./Tasks/Tasks";

const TasksWrapper = styled.section`
  width: 100%;
  min-height: 100vh;
  padding: 5rem 0 2rem 0;
  position: relative;

  &:after {
    content: "";
    width: 100%;
    height: 5rem;
    background: linear-gradient(
      180deg,
      rgba(78, 76, 165, 1) 50%,
      rgba(62, 61, 132, 1) 100%
    );
    position: absolute;
    top: 0;
    left: 0;
    z-index: 1;
  }
`;
const TasksSection: React.FC = () => {
  return (
    <TasksWrapper>
      <Container maxWidth="xl" sx={{ padding: { xs: 0, md: "0 24px 0 24px" } }}>
        <ContentPoint index="02" heading="Organize time" />
        <Grid container sx={{ marginTop: 5 }}>
          <Grid item xs={12} md={6}>
            <Typography
              variant="h5"
              sx={{
                textAlign: "center",
                fontFamily: "var(--font-heading)",
              }}
            >
              Organize your time with perfect Tasks List
            </Typography>
          </Grid>
          <Grid item xs={12} md={6}>
            <Tasks />
          </Grid>
        </Grid>
      </Container>
    </TasksWrapper>
  );
};

export default TasksSection;
