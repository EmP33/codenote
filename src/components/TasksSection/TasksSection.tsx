import React from "react";
import { Container, Grid, Typography } from "@mui/material";
import styled from "styled-components";
import ContentPoint from "../ContentPoint/ContentPoint";

const NotesWrapper = styled.section`
  width: 100%;
  height: 100vh;
  padding: 5rem 0 0 0;
  position: relative;

  &:after {
    content: "";
    width: 100%;
    height: 5rem;
    background: linear-gradient(
      180deg,
      rgba(19, 15, 50, 1) 50%,
      rgba(62, 61, 132, 1) 100%
    );
    position: absolute;
    top: 0;
    left: 0;
    z-index: 1;
  }
`;
const TasksSection: React.FC = () => {
  return (
    <NotesWrapper>
      <Container maxWidth="xl">
        <ContentPoint index="02" heading="Organize time" />
        <Grid container>
          <Grid item>
            <Typography
              variant="h5"
              sx={{ textAlign: "center", fontFamily: "var(--font-heading)" }}
            >
              Organize your time with perfect Tasks List
            </Typography>
          </Grid>
        </Grid>
      </Container>
    </NotesWrapper>
  );
};

export default TasksSection;
