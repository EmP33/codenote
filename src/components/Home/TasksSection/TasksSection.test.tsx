import { shallow } from "enzyme";
import TasksSection from "./TasksSection";

it("should render TasksSection properly", () => {
  expect(shallow(<TasksSection />)).toMatchSnapshot();
});
