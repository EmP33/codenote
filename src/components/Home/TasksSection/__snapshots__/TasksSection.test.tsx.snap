// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TasksSection properly 1`] = `
<styled.section>
  <ForwardRef(Container)
    maxWidth="xl"
    sx={
      Object {
        "padding": Object {
          "md": "0 24px 0 24px",
          "xs": 0,
        },
      }
    }
  >
    <ContentPoint
      heading="Organize time"
      index="02"
    />
    <ForwardRef(Grid)
      container={true}
      sx={
        Object {
          "marginTop": 5,
        }
      }
    >
      <ForwardRef(Grid)
        item={true}
        md={6}
        xs={12}
      >
        <ForwardRef(Typography)
          sx={
            Object {
              "fontFamily": "var(--font-heading)",
              "textAlign": "center",
            }
          }
          variant="h5"
        >
          Organize your time with perfect Tasks List
        </ForwardRef(Typography)>
      </ForwardRef(Grid)>
      <ForwardRef(Grid)
        item={true}
        md={6}
        xs={12}
      >
        <Tasks />
      </ForwardRef(Grid)>
    </ForwardRef(Grid)>
  </ForwardRef(Container)>
</styled.section>
`;
